packages/@okta/vuepress-site/docs/guides/add-an-external-idp/use-idp-to-sign-in/index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Use the Identity Provider to sign in
+---
+To test your authorization URL, enter the complete authorization URL in a browser. Do this in your browser's privacy or incognito mode to avoid false positive or negative results.
+
+If everything is configured properly:
+
+* The user is redirected to the Identity Provider's sign-in page.
+* After successful authentication, the user is redirected to the redirect URI that you specified, along with an `#id_token=` fragment in the URL. The value of this parameter is your Okta OpenID Connect ID token.
+
+If something is configured incorrectly, the authorization response contains error information to help you resolve the issue.
+
+There are four primary ways to kick off the sign-in flow.
+
+**HTML Link**
```

Comment:
Should the `HTML Link`, `Custom Okta-hosted Sign-in Page`, etc sections have real headings?

That is, `##` or `###` H2s or H3s? Then they are easier to see, can be linked to, will appear in (future) on-page table of contents, etc.

Reply:
Do we have a rationale for the order in which these sections appear? It might be good to put them in order of likelihood.

Do we know the order of likelihood? I would expect the Sign-in Widget to be one of the more common methods, especially early in a developer's journey with Okta.

Reply:
Eventually we probably also want to provide guidance for native UI in native iOS and Android apps. That is, creating a button in a native iOS or Android view, _not_ showing the widget in a native app.

Is there any guidance we can provide for that scenario today?

Reply:
@alexdahl-okta

There isn't AFAIK. We should probably open a JIRA to address this in the future, so we don't forget.

Reply:
> Eventually we probably also want to provide guidance for native UI in native iOS and Android apps. That is, creating a button in a native iOS or Android view, _not_ showing the widget in a native app.
> 
> Is there any guidance we can provide for that scenario today?

@alexdahl-okta
There isn't AFAIK. We should probably open a JIRA to address this in the future, so we don't forget.

Reply:
> Do we have a rationale for the order in which these sections appear? It might be good to put them in order of likelihood.
> 
> Do we know the order of likelihood? I would expect the Sign-in Widget to be one of the more common methods, especially early in a developer's journey with Okta.

My expectations would be html link first, especially if they are just testing....

And then the widget, custom sign-in page, and AuthJS - but we should get input from both @albertchen-okta and @venkatviswanathan-okta